Unable to build QuantityUGens

Current master is broken for me on Linux. The problem seems to be this line in the recently added **QuantitiyUGens**:
https://github.com/supercollider/sc3-plugins/blob/7fb101b51c87f6b69062c20d6c32cf63a23381ab/source/QuantityUGens/QuantityUGens.cpp#L82

The error I get when building is:

 ```
/sc3-plugins/source/QuantityUGens/QuantityUGens.cpp: In function 'void MovingXCtorHelper(MovingX*, UnitCalcFunc)':
/sc3-plugins/source/QuantityUGens/QuantityUGens.cpp:82:9: error: 'printf' was not declared in this scope
        printf("MovingSum/Average Error:\n\t'maxsamp' argument must be >= 1, and within integer resolution.\n\tReceived: %f\n", ZIN0(2));
        ^~~~~~
/sc3-plugins/source/QuantityUGens/QuantityUGens.cpp:82:9: note: suggested alternative: 'rintf'
        printf("MovingSum/Average Error:\n\t'maxsamp' argument must be >= 1, and within integer resolution.\n\tReceived: %f\n", ZIN0(2));
        ^~~~~~
        rintf
source/CMakeFiles/QuantityUGens_supernova.dir/build.make:62: recipe for target 'source/CMakeFiles/QuantityUGens_supernova.dir/QuantityUGens/QuantityUGens.cpp.o' failed
make[2]: *** [source/CMakeFiles/QuantityUGens_supernova.dir/QuantityUGens/QuantityUGens.cpp.o] Error 1
CMakeFiles/Makefile2:3208: recipe for target 'source/CMakeFiles/QuantityUGens_supernova.dir/all' failed
make[1]: *** [source/CMakeFiles/QuantityUGens_supernova.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/source/QuantityUGens/QuantityUGens.cpp
+++ b/source/QuantityUGens/QuantityUGens.cpp
@@ -79,7 +79,7 @@ inline void MovingXCtorHelper( MovingX* unit, UnitCalcFunc calcFunc )
     unit->msquares = nullptr; // in case we error out before msquares is assigned
     
     if (ZIN0(2) < 1 || ZIN0(2) > std::numeric_limits<int>::max() || !std::isfinite(ZIN0(2))) {
-        printf("MovingSum/Average Error:\n\t'maxsamp' argument must be >= 1, and within integer resolution.\n\tReceived: %f\n", ZIN0(2));
+        Print("MovingSum/Average Error:\n\t'maxsamp' argument must be >= 1, and within integer resolution.\n\tReceived: %f\n", ZIN0(2));
         SETCALC(*ClearUnitOutputs);
         ClearUnitOutputs(unit, 1);
         unit->mDone = true;

